conformance/resources/manifests/manifests.yaml

```diff
@@ -66,7 +54,95 @@ spec:
   - name: http
     port: 80
     protocol: HTTP
-    hostname: "secondary.example.com" # Distinct hostname to differentiate from conformance-gateway
+    hostname: "secondary.example.com"
     allowedRoutes:
       namespaces:
         from: All
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: inference-model-1-deployment
```

Comment:
nit: Drop the resource type as a suffix: s/inference-model-1-deployment/inference-model-1/g

Reply:
Done.